k8s/aat/cluster-00-overlay/rd/kustomization.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-namespace: rd
-bases:
-- ../../../namespaces/rd/rd-judicial-data-load/rd-judicial-data-load.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/rd-location-ref-data-load.yaml
-patchesStrategicMerge:
-- ../../../namespaces/rd/rd-judicial-data-load/aat-00.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/aat-00.yaml
\ No newline at end of file
+    kind: Kustomization
```

Comment:
can you revert these pls

Reply:
Thanks blaming Inteliji on that one, hopefully this is better now!